oak-auth-external/src/main/java/org/apache/jackrabbit/oak/spi/security/authentication/external/impl/principal/ExternalPrincipalConfiguration.java

```diff
@@ -78,6 +84,14 @@
                 label = "External Identity Protection",
                 description = "If disabled rep:externalId properties won't be properly protected (backwards compatible behavior). NOTE: for security reasons it is strongly recommend to keep the protection enabled!",
                 boolValue = ExternalIdentityConstants.DEFAULT_PROTECT_EXTERNAL_IDS),
+        @Property(name = PARAM_PROTECT_EXTERNAL_IDENTITIES,
+                label = "External User and Group Protection",
+                description = "If 'None' is selected the synchronized external users/groups won't be protected (backwards compatible behavior) and can be edited like local users/groups. NOTE: in order to avoid having inconsistencies between the IDP that defines the external identities and local synced identities it is recommend to enable the protection. With option 'Warn' the protection is disabled by warnings will be logged.",
```

Comment:
There seems to be a typo in the description:

With option 'Warn' the protection is disabled **but** warnings will be logged

Reply:
will fix it together with the other changes we discussed today.